### ISSUE TITLE
Error: fft: ATen not compiled with FFT support

### ISSUE THREAD

**Issue by LeonKennedy** [pytorch/live#62]
### Version

0.1

### Problem Area

react-native-pytorch-core (core package)

### Steps to Reproduce

 I have a model trained by myself in python(Unix). but it can't be running in react size(Macos). 

output:
 `Possible Unhandled Promise Rejection (id: 2):
Error: fft: ATen not compiled with FFT support

  Debug info for handle(s): -1, was not found.

Exception raised from _fft_r2c_mkl at ../aten/src/ATen/native/mkl/SpectralOps.cpp:569 (most recent call first):
(no backtrace available)
Error: fft: ATen not compiled with FFT support

  Debug info for handle(s): -1, was not found.

Exception raised from _fft_r2c_mkl at ../aten/src/ATen/native/mkl/SpectralOps.cpp:569 (most recent call first):
(no backtrace available)
    at promiseMethodWrapper (http://localhost:8081/index.bundle?platform=android&dev=true&minify=false&app=org.pytorch.live.example&modulesOnly=false&runModule=true:2772:45)
    at _callee3$ (http://localhost:8081/index.bundle?platform=android&dev=true&minify=false&app=org.pytorch.live.example&modulesOnly=false&runModule=true:109141:76)
    at call (native)
    at tryCatch (http://localhost:8081/index.bundle?platform=android&dev=true&minify=false&app=org.pytorch.live.example&modulesOnly=false&runModule=true:7703:23)
    at invoke (http://localhost:8081/index.bundle?platform=android&dev=true&minify=false&app=org.pytorch.live.example&modulesOnly=false&runModule=true:7876:32)
    at anonymous (http://localhost:8081/index.bundle?platform=android&dev=true&minify=false&app=org.pytorch.live.example&modulesOnly=false&runModule=true:7746:30)
    at call (native)
    at tryCatch (http://localhost:8081/index.bundle?platform=android&dev=true&minify=false&app=org.pytorch.live.example&modulesOnly=false&runModule=true:7703:23)
    at invoke (http://localhost:8081/index.bundle?platform=android&dev=true&minify=false&app=org.pytorch.live.example&modulesOnly=false&runModule=true:7776:30)
    at anonymous (http://localhost:8081/index.bundle?platform=android&dev=true&minify=false&app=org.pytorch.live.example&modulesOnly=false&runModule=true:7806:19)
    at tryCallTwo (http://localhost:8081/index.bundle?platform=android&dev=true&minify=false&app=org.pytorch.live.example&modulesOnly=false&runModule=true:28581:9)
    at doResolve (http://localhost:8081/index.bundle?platform=android&dev=true&minify=false&app=org.pytorch.live.example&modulesOnly=false&runModule=true:28745:25)
    at Promise (http://localhost:8081/index.bundle?platform=android&dev=true&minify=false&app=org.pytorch.live.example&modulesOnly=false&runModule=true:28604:14)
    at callInvokeWithMethodAndArg (http://localhost:8081/index.bundle?platform=android&dev=true&minify=false&app=org.pytorch.live.example&modulesOnly=false&runModule=true:7805:33)
    at enqueue (http://localhost:8081/index.bundle?platform=android&dev=true&minify=false&app=org.pytorch.live.example&modulesOnly=false&runModule=true:7810:157)
    at anonymous (http://localhost:8081/index.bundle?platform=android&dev=true&minify=false&app=org.pytorch.live.example&modulesOnly=false&runModule=true:7746:30)
    at anonymous (http://localhost:8081/index.bundle?platform=android&dev=true&minify=false&app=org.pytorch.live.example&modulesOnly=false&run`

**Comment by LeonKennedy** [comment]
Train Side:
System: Ubantu 20.04
Cuda: 10.2
cudnn: 7.6
pytorch: 1.11
model: ASR(Enformer+CTC) 
feature_extract: torchaudio.transforms.MelSpectrogram(n_mels=80, hop_length=160) 

`
>>> torch.__config__.show()
'PyTorch built with:\n  - GCC 7.3\n  - C++ Version: 201402\n  - Intel(R) Math Kernel Library Version 2020.0.0 Product Build 20191122 for Intel(R) 64 architecture applications\n  - Intel(R) MKL-DNN v2.5.2 (Git Hash a9302535553c73243c632ad3c4c80beec3d19a1e)\n  - OpenMP 201511 (a.k.a. OpenMP 4.5)\n  - LAPACK is enabled (usually provided by MKL)\n  - NNPACK is enabled\n  - CPU capability usage: AVX2\n  - CUDA Runtime 10.2\n  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_70,code=sm_70\n  - CuDNN 7.6.5\n  - Magma 2.5.2\n  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=10.2, CUDNN_VERSION=7.6.5, CXX_COMPILER=/opt/rh/devtoolset-7/root/usr/bin/c++, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_KINETO -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DSYMBOLICATE_MOBILE_DEBUG_HANDLE -DEDGE_PROFILER_USE_KINETO -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_VERSION=1.11.0, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=OFF, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_ROCM=OFF, \n'
`

export model code:

`
 model, loss = get_live_model("model_l8.pt", 8)
 model = torch.quantization.convert(model)
 scripted_model = torch.jit.script(model)
 opt_model = optimize_for_mobile(scripted_model)
 dataset = VoiceTestData("/mnt/data/conformer/test.tsv", num=10)
 for audio, origin, phonemes, wav_filename in dataset:
     target = opt_model(torch.Tensor(audio))
     print([ELEMENTS[i] for i in target.tolist() if i != 0], origin, phonemes)

 spec = Path("live.spec.json").read_text()
 extra_files = {}
 extra_files["model/live.spec.json"] = spec
 opt_model._save_for_lite_interpreter("mobile_model.ptl",  _extra_files=extra_files)
`

**Comment by LeonKennedy** [comment]
React Live Side:
System: Macos Monterey 12.3.1

i follow the instruction (https://pytorch.org/live/docs/tutorials/get-started/) in Android. 
i can run every example demo success, like IMAGE CLASSIFICATION, QUESTION ANSWERING and Wav2vec2.

**Comment by LeonKennedy** [comment]
So which side has the problem?(fft: ATen not compiled with FFT support) Unix(train)?Macos(inference)? How could i fix it? thanks!

**Comment by raedle** [comment]
@LeonKennedy, PyTorch Mobile doesn't support MKL. See the discussion in the PyTorch GitHub repo:

https://github.com/pytorch/pytorch/issues/29796

Closing issue because this is outside the purview of PyTorch Live
